spec/system/submissions/review_spec.rb

```diff
@@ -1405,5 +1410,84 @@
       expect(page).to_not have_text(team_1.name)
       expect(page).to_not have_text(team_2.name)
     end
+
+    context 'when a submission report exists for a submission' do
```

Comment:
We should also check the other case 'when a report doesn't exist all UI elements related to the report should be hidden`.